client/src/components/header/header.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './header.css';
+
+
+export class Header extends React.Component {
+    state = {
+        slideClass: "nav-links",
+        animate: "",
+        burger: "burger"
+    }
+    ToggleSlide = () => {
```

Comment:
I haven't seen all your code but there could be a way to refactor this

Reply:
Please provide more details and link the wanted-to-refactor lines